backend/src/authentication/authentication.service.ts

```diff
@@ -26,11 +30,30 @@ export class AuthenticationService {
         return `${registrationData.email} ${content.Created}`;
     }
 
+    /**
+      Currently this returns the pin as we need pass that back to frontend.
+      Will be corrected when relevant workflow is introduced.
+     */
+    async registerJunior(registrationData: RegisterJuniorDto): Promise<string> {
```

Comment:
Should this method be in juniorService?

Reply:
Currently I would argue that all login and registration methods (e.g. creating accounts  and generating JWT tokens) are best moved into their own service, as this will help the developers ensure consistency between such methods.